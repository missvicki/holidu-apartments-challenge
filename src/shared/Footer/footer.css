footer {
  position: fixed;
  width: 100%;
  bottom: 0;
  background: white;
}

.footer__content {
  height: 80px;
  border-top: 1px solid #ededed;
  display: flex;
  justify-content: center;
  align-items: center;
  font-size: 12px;
}
