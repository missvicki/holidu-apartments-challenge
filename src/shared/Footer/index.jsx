import React from 'react';
import './footer.css';

const Footer = () => (
  <footer>
    <div className="footer__content" data-testid="footer">
      &copy; Holidu Apartments
    </div>
  </footer>
);

export default Footer;
