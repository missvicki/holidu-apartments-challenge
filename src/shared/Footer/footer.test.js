import React from 'react';
import { shallow } from 'enzyme';
import Footer from './index';

describe('Footer', () => {
  it('should not have regressed', () => {
    const component = shallow(<Footer />);

    expect(component).toMatchSnapshot();
  });
});
