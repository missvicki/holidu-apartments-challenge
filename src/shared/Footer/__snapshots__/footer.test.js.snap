// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should not have regressed 1`] = `
<div
  className="footer"
>
  <h2
    className="foot"
  >
    Copyright Â© 2020
  </h2>
</div>
`;
